README.md

```diff
@@ -466,6 +466,60 @@ Simbody is installed into the `usr/` directory.  The directory
     * `examples/src` source code for the examples.
     * `examples/bin` symbolic link to executable examples.
 
+Windows using MinGW
+-------------------
+
+Warning: The [MinGW](http://sourceforge.net/projects/mingw-w64/)
+generation and build is experimental!
+
+This build is still experimental, because of the precompiled libraries
+Simbody depends on (Blas and Lapack).
+To ease the compilation Windows, Simbody provides precompiled libraries for Blas and Lapack.
```

Comment:
This sentence has a typo: "To ease the compilation Windows". Perhaps "To ease building on Windows"?

Also, "precompiled libraries" makes me think of "precompiled headers". Wouldn't it be sufficient to say "compiled libraries"?